cpp/src/Ice/SSL/OpenSSLTransceiverI.cpp

```diff
@@ -36,10 +36,13 @@ extern "C"
 
 namespace
 {
-    std::function<bool(bool, X509_STORE_CTX* ctx, const Ice::SSL::ConnectionInfoPtr&)>
-    createDefaultVerificationCallback()
+    bool defaultVerificationCallback(bool ok, X509_STORE_CTX*, const Ice::SSL::ConnectionInfoPtr&) { return ok; }
```

Comment:
This avoids creating one callback per connection when the user doesn't provide one.